airbyte-integrations/connectors/destination-jdbc/src/main/java/io/airbyte/integrations/destination/jdbc/copy/s3/S3StreamCopier.java

```diff
@@ -48,21 +48,21 @@
   // WARNING: Too large a part size can cause potential OOM errors.
   public static final int DEFAULT_PART_SIZE_MB = 10;
 
-  private final AmazonS3 s3Client;
-  private final S3Config s3Config;
-  private final String tmpTableName;
+  protected final AmazonS3 s3Client;
+  protected final S3Config s3Config;
+  protected final String tmpTableName;
   private final DestinationSyncMode destSyncMode;
-  private final String schemaName;
-  private final String streamName;
-  private final JdbcDatabase db;
+  protected final String schemaName;
+  protected final String streamName;
+  protected final JdbcDatabase db;
   private final ExtendedNameTransformer nameTransformer;
   private final SqlOperations sqlOperations;
-  private final Set<String> s3StagingFiles = new HashSet<>();
+  protected final Set<String> s3StagingFiles = new HashSet<>();
```

Comment:
This was probably the only variable that definitely needed to be changed to `protected` for child class access, but I changed the others because seemed cleaner than defining them all again in: `RedshiftStreamCopier.java`